pagopa/pn-frontend
# Moved privacy policy page to cittadini one - Feat/pn 7866

### MERGE AFTER PN-7865

## Short description
Now the login application is hosted under cittadini.notifichedigitali.it, so the dedicated privacy policy page is useless.

## List of changes proposed in this pull request
- Removed dedicated privacy policy page and all configuration
- Link the privacy policy to the page under cittadini.notifichedigitali.it

## How to test
Go to pf login and click on privacy policy link -> check that the user is redirect under cittadini.{env}.notifichedigitali.it/informativa-privacy
